docs/install.md

```diff
@@ -80,7 +99,8 @@ sure you have [configured `kubectl` to access your cluster](https://kubernetes.i
    (see
    [AWS Security Group rules](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_SecurityGroups.html),
    [Google Cloud Platform Firewall rules](https://cloud.google.com/compute/docs/vpc/using-firewalls)). Sourcegraph
-   should then be accessible at `$EXTERNAL_ADDR_OF_YOUR_NODE:30080`. For production environments, we recommend using
+   should then be accessible at `$EXTERNAL_ADDR_OF_YOUR_NODE:30080`. If you have multiple nodes, all the addresses
```

Comment:
Change to "should then be accessible at `$EXTERNAL_NODE_ADDR:30080`, using the external address of *any* of the nodes in the cluster."

No need to mention the implementation detail of `kube-proxy` here.